guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -12,6 +12,12 @@ include::snip_firewalld.adoc[]
 ----
 # yum install postgresql-server
 ----
+
+. Install the `rh-postgresql12-postgresql-evr` package:
++
+----
+# yum install rh-postgresql12-postgresql-evr
```

Comment:
Note that we only build this package on EL7 now so the host must be EL7. Upstream we do build it for EL8 where it's named `postgresql-evr`. I don't know about downstream.

Reply:
Downstream we say that the external database must be installed on RHEL7, so keeping the EL7 name works fine.